src/vec_bcd_ppc.h

```diff
@@ -1423,13 +2481,24 @@ vec_bcdsubcsq (vBCD_t a, vBCD_t b)
 {
   vBCD_t t;
 #if defined (_ARCH_PWR8) && (__GNUC__ > 6)
+  vBCD_t a_b;
+#ifdef _ARCH_PWR9
+  // Generate BCD zero from (a - a), which is 3 cycles on PWR9
+  t = vec_bcdsub (a,  a);
+#else // Else load a BCD const 0.
   t = _BCD_CONST_ZERO;
+#endif
+  a_b = vec_bcdsub (a, b);
   if (__builtin_expect (__builtin_bcdsub_ov ((vi128_t) a, (vi128_t) b, 0), 0))
     {
+#ifdef _ARCH_PWR9
+      t = vec_bcdcpsgn (_BCD_CONST_PLUS_ONE, a_b);
+#else
       if (__builtin_bcdsub_gt ((vi128_t) a, (vi128_t) b, 0))
-	t = _BCD_CONST_PLUS_ONE;
+      t = _BCD_CONST_PLUS_ONE;
```

Comment:
not sure the indentation is right here

Reply:
Yea, This seems to be another case where the SDK GNU format tool not formating correctly